### ISSUE TITLE
Training set 

### ISSUE THREAD

**Issue by clelouch** [guyuchao/PyramidCSA#10]
Thanks for your code and paper.
Did you merge the validation set and the training set of the DAVSOD for training?
Besides, the PyramidCSA uses the DAVIS as the training set while test on it. Did you split the DAVIS into a train subset and a test subset?

**Comment by guyuchao** [comment]
The DAVIS had it own train and val split. We use the official train/val split. We use the training set of DAVSOD only.
